doc/classes/Tween.xml

```diff
@@ -207,6 +207,7 @@
 			<description>
 				Pauses the tweening. The animation can be resumed by using [method play].
 				[b]Note:[/b] If a Tween is paused and not bound to any node, it will exist indefinitely until manually started or invalidated. If you lose a reference to such Tween, you can retrieve it using [method SceneTree.get_processed_tweens].
+				[b]Note:[/b] When a subtween (set via [method tween_subtween]) is paused, it will hold up execution of the parent [Tween] on that step. Make sure to either resume the subtween with [method play] or end it with [method kill] so that the rest of the parent [Tween] can finish executing.
```

Comment:
I wonder if we should "pollute" method description like this or just add some more notes to `tween_subtween()` instead.